torch_geometric/nn/models/jumping_knowledge.py

```diff
@@ -40,13 +43,14 @@ def __init__(self, mode, channels=None, num_layers=None):
         super().__init__()
         self.mode = mode.lower()
         assert self.mode in ['cat', 'max', 'lstm']
+        # Must declare these here in order to use torch.jit.script
+        self.lstm = LSTM(channels, (num_layers * channels) // 2,
```

Comment:
Why is that? We are doing something similar in [SAGEConv](https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/nn/conv/sage_conv.py#L99) and it is TorchScript compatible. I think the key is to have some additional variable that denotes if this module should be applied.

Reply:
When it's declared inside of the condtional I get the following error:
`Module 'JumpingKnowledge' has no attribute 'lstm'`
I've changed:
`if mode == 'lstm':`
to
`if self.mode == 'lstm':`
but that doesn't seem to fix it.

Reply:
Ahh, yes. This makes sense. Thanks!